README.md

```diff
@@ -127,12 +131,148 @@ The recommended way to use the AWS Glue Schema Registry Library for Java is to c
+                                                + "              \"age\",\n" + "              \"city\"\n" + "            ]\n" + "          }\n"
+                                                + "        },\n" + "        \"required\": [\n" + "          \"employee\"\n" + "        ]\n"
+                                                + "      }";
+        String jsonPayload = "{\n" + "        \"employee\": {\n" + "          \"name\": \"John\",\n" + "          \"age\": 30,\n"
+                                                 + "          \"city\": \"New York\"\n" + "        }\n" + "      }";
+        
+        JsonDataWithSchema jsonSchemaWithData = JsonDataWithSchema.builder(jsonSchema, jsonPayload);
+
+        List<JsonDataWithSchema> genericJsonRecords = new ArrayList<>();
+        genericJsonRecords.add(jsonSchemaWithData);
+        
+        try (KafkaProducer<String, JsonDataWithSchema> producer = new KafkaProducer<String, JsonDataWithSchema>(properties)) {
+            for (int i = 0; i < genericJsonRecords.size(); i++) {
+                JsonDataWithSchema r = genericJsonRecords.get(i);
+
+                final ProducerRecord<String, JsonDataWithSchema> record;
+                record = new ProducerRecord<String, JsonDataWithSchema>(topic, "message-" + i, r);
+
+                producer.send(record);
+                System.out.println("Sent message " + i);
+                Thread.sleep(1000L);
+            }
+            producer.flush();
+            System.out.println("Successfully produced 10 messages to a topic called " + topic);
+
+        } catch (final InterruptedException | SerializationException e) {
+            e.printStackTrace();
+        }
+```
+
+#### Consumer for Kafka with JSON format
+
+```java
+        properties.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
+        properties.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, GlueSchemaRegistryKafkaDeserializer.class
+        .getName();
+        properties.put(AWSSchemaRegistryConstants.AWS_REGION, "us-east-1");
+        
+        try (final KafkaConsumer<String, JsonDataWithSchema> consumer = new KafkaConsumer<String, JsonDataWithSchema>(properties)) {
+            consumer.subscribe(Collections.singletonList(topic));
+
+            while (true) {
+                final ConsumerRecords<String, JsonDataWithSchema> records = consumer.poll(100);
+                for (final ConsumerRecord<String, JsonDataWithSchema> record : records) {
+                    final String key = record.key();
+                    final JsonDataWithSchema value = record.value();
+                    System.out.println("Received message: key = " + key + ", value = " + value);
+                }
+            }
+        }
+
+```
+
+### Dealing with Specific Record (JAVA POJO) for JSON
+
+You could use a Java POJO and pass the object as a record. This is similar to the notion of a Specific Record in AVRO.
```

Comment:
The idea of Specific record might be new to JSON only users. We can drop it and just say support for POJO

Reply:
Removed